include/ray.h

```diff
@@ -9,20 +9,28 @@ class Ray {
 private:
   Vertex origin_;
   Direction direction_;
+  float importance_;
```

Comment:
Yeah, but it pretty much works the same way now. And we need importance later anyway, depth isn't really necessary if importance drops anyway!